gratipay/models/participant/email.py

```diff
@@ -206,8 +215,8 @@ def remove_email(self, address):
 
 
     def queue_email(self, spt_name, **context):
```

Comment:
> some

That was an understatement. :-)

With #4350 and #4355, we now have queueing and flushing in the same class.